GetStream/stream-chat-android
# [CAS-1072] Reorganize client docs

### 🎯 Goal

Reorganize client docs

### 🧪 Testing

1. Run `npx stream-chat-docusaurus -i -s`
2. Check `Client` section

### ☑️ Checklist
